AppleALC/kern_alc.cpp

```diff
@@ -16,10 +16,19 @@
 #include "kern_resources.hpp"
 
 // Only used in apple-driven callbacks
-static AlcEnabler *callbackAlc;
+AlcEnabler* AlcEnabler::callbackAlc = nullptr;
+
+void AlcEnabler::createShared() {
+	if (callbackAlc)
+		PANIC("alc", "Attempted to allocated AlcEnabler again");
+	
+	callbackAlc = new AlcEnabler;
```

Comment:
Any particular reason it needs to become dynamically initialised out of a sudden?

Reply:
Nothing specific. I need it so that I can be sure that there is only one instance created, on demand. Also, makes sharing instance across project eloquent.

Reply:
Probably easier to allocate it statically and just assign here to avoid dynamic allocations for no reason. Also, Attempted to allocated → Attempted to allocate.